install.md

```diff
@@ -22,68 +22,27 @@
 Nmstate is in Fedora and EPEL 7 testing, you may install it using below
 commands.
 
- * Fedora 28+
+ * Fedora 31+
 ```bash
 sudo dnf install nmstate
-```
-
- * RHEL 7 using EPEL7-testing.
-
-```bash
-sudo subscription-manager repos --enable \
-    "rhel-*-optional-rpms" --enable "rhel-*-extras-rpms"
-sudo yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
-sudo yum-config-manager --enable epel-testing
-sudo yum install nmstate
-```
-
- * Centos 7 using EPEL7-testing.
-
-```bash
-sudo yum install epel-release -y
-sudo yum-config-manager --enable epel-testing
-sudo yum install nmstate
-```
-
- * RHEL/Centos 7 using copr repo
-
-```bash
-sudo yum install yum-plugin-copr
-sudo yum copr enable nmstate/nmstate-el7
-sudo yum install nmstate
 ```
 
  * RHEL 8 using copr repo
 
 ```bash
-sudo dnf copr enable nmstate/nmstate-el8
+sudo dnf copr enable nmstate/nmstate-stable
 sudo dnf install nmstate
 ```
 
 ### Developer Branch
 We have copr repos which automatically build whenever a patch goes into
 git master branch. Only for develop use.
 
- * Fedora 28+
+ * Fedora 31+, CentOS/RHEL 8
```

Comment:
Is copr relevant for centos and rhel?

Reply:
Why not?

Reply:
@EdDev can you please elaborate? On the mailing list you specifically asked for Copr repos for CentOS, so it seems to be relevant.

Reply:
IMHO, if we have Copr repos for CentOS then it is relevant.